draft-tuexen-opsawg-pcapng.xml

```diff
@@ -1828,6 +1843,168 @@ Section Header
+          <ttcol align="left">Name</ttcol>
+          <ttcol align="left">Code</ttcol>
+          <ttcol align="left">Length</ttcol>
+          <ttcol align="left">Multiple allowed?</ttcol>
+
+	        <c>interface_id</c>
+          <c>1</c>
+          <c>4</c>
+          <c>no</c>
+
+          <c>global_location</c>
+          <c>2</c>
+          <c>12</c>
+          <c>no</c>
+
+          <c>global_location_error</c>
+          <c>3</c>
+          <c>12</c>
+          <c>no</c>
+
+          <c>orientation</c>
+          <c>4</c>
+          <c>12</c>
+          <c>no</c>
+
+          <c>orientation_error</c>
+          <c>5</c>
+          <c>12</c>
+          <c>no</c>
+
+          <c>velocity</c>
+          <c>6</c>
+          <c>12</c>
+          <c>no</c>
+
+          <c>velocity_error</c>
+          <c>7</c>
+          <c>12</c>
+          <c>no</c>
+        </texttable>
+
+        <t>
+          <list hangIndent="8" style="hanging">
+
+            <t hangText="cib_interface_id:"><vspace blankLines="0"/>The
+	          interface_id option specifies the interface these statistics refers
+            to; the correct interface will be the one whose Interface
+            Description Block (within the current Section of the file) is
+            identified by same number (see <xref target="section_idb"/>) of
+            this field.</t>
+
+	          <t hangText="cib_global_location"><vspace blankLines="0"/>The 
+            cib_global_location option specifies the location of the packet 
+            capture or interface; location is stored as three 32-bit floats 
+            representing latitude and longitude in degrees, and altitude in 
+            metres.</t>
+
+            <t>Example: TODO</t>
+
+            <t hangText="cib_global_location_error"><vspace blankLines="0"/>The
+            global_location_error option specifies the error in location capture; 
+            this is stored as three 32-bit floats in the same order and 
+            representation as cib_location.</t>
+
+            <t>Example: TODO</t>
+
+            <t hangText="cib_orientation"><vspace blankLines="0"/>The 
+            cib_orientation option specifies the location of the packet 
+            capture or interface; location is stored as three 32-bit floats 
+            representing yaw, pitch, and roll in radians.</t>
+
+            <t>Example: TODO</t>
+
+            <t hangText="cib_orientation_error"><vspace blankLines="0"/>The
+            orientation_error option specifies the error in orientation capture; 
+            this is stored as three 32-bit floats in the same order and 
+            representation as cib_orientation.</t>
+
+            <t>Example: TODO</t>
+
+            <t hangText="cib_velocity"><vspace blankLines="0"/>The 
+            cib_velocity option specifies the location of the packet 
+            capture or interface; location is stored as three 32-bit floats 
```

Comment:
"*velocity* is stored as three 32-bit floats representing *the rates of change of latitude and longitude in degrees per second, and the rate of change of altitude in metres/second*."

Reply:
I think it makes more sense for velocity to be in meters per second against a global reference frame (orientation zero, Y facing north, -Z down). It's easier to compare and use in calculations, and works for non-global locations (see PR update introducing local locations too).